samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor

```diff
@@ -38,7 +38,9 @@
 
 @if (EndpointUrl != null)
 {
-    <p>Url: @EndpointUrl</p>
+    <div class="subtitle">
```

Comment:
where the subtitle class is defined?

Reply:
In site.css. It has been defined in one of my previous PRs